initfiles/bash/etc/init.d/install-init.in

```diff
@@ -197,6 +197,12 @@ installFile "$binPath/eclplus" "/usr/bin/eclplus" 1 || exit 1
 installFile "$binPath/wuget" "/usr/bin/wuget" 1 || exit 1
 installFile "$configs/$environment" "$sourcedir/$environment" 0 "$sourcedir"
 
+installFile "$path${CONFIG_DIR}/configmgr/configmgr.conf"  \
+     "${CONFIG_DIR}/configmgr/" 0 "${CONFIG_DIR}/configmgr" || exit 1
+installFile "$path${CONFIG_DIR}/configmgr/esp.xml" "${CONFIG_DIR}/configmgr/" 0 || exit 1
+installFile "$path${CONFIG_DIR}/configmgr/config2mgr.conf"  \
```

Comment:
After testing, I don't see this file laid down.  It generates the /etc/config2mgr directory and remains empty. Looking through the source I wasn't finding a reference to a config2mgr.conf file anywhere.  Has that been implemented yet?

Reply:
Remove config2mgr.conf and add esp.xml